tests/unit/test_zero.py

```diff
@@ -28,7 +28,7 @@ def enable_grads(model):
         enable_grads(model)
 
 
-@pytest.mark.parametrize('zero_stage', [1, 2])
+@pytest.mark.parametrize('zero_stage', [1, 2, 3])

```

Comment:
added stage 3 here as well - hope it's ok

Reply:
Sure!